tests/syslog/test_syslog.py

```diff
@@ -16,26 +16,25 @@
 SYSLOG_MESSAGE_TEST_TIMEOUT = 10
 
 
-@pytest.fixture(scope="module")
-def config_syslog_srv(ptfhost, duthost):
+def config_syslog_srv(ptfhost):
```

Comment:
Is there anything we should do for clean-up after the syslog tests are finished?

Reply:
No. that is why I changed to a helper function.